private-dns-config.yml

```
@@ -10,7 +10,7 @@ subscriptions:
     resourceGroup: core-infra-intsvc-rg
 
   - name: "DTS-CFTPTL-INTSVC"
-    zoneTemplate: '${["plum","probate","sscs","idam","ccd","ts","hmc","aac","rpo","camunda","dg","dm","dtsse","em","et","sptribs","nfdiv","pcq","lau","wa","finrem","jps","rd","xui", "reform-scan", "bulk-scan", "rpe", "fact", "rpts", "am", "ia"].contains(product) || ["draft-store-service","div-cos","civil-general-applications", "civil-service", "civil-ccd","civil-camunda","civil-citizen-ui","civil-ga-ccd","rpe-service-auth-provider", "rpe-pdf-service", "cmc-citizen-frontend", "nfdiv-case-api", "nfdiv-frontend","prl-ccd-definitions","prl-cos","prl-dgs","prl-citizen-frontend","adoption-cos-api","adoption-web","ccd-case-disposer", "fpl-case-service"].contains(appFullName) ? environment + ".platform.hmcts.net" : "service.core-compute-" + environment + ".internal"}'
+    zoneTemplate: '${["plum","probate","sscs","idam","ccd","ts","hmc","aac","rpo","camunda","dg","dm","dtsse","em","et","sptribs","nfdiv","pcq","lau","wa","finrem","jps","rd","xui", "reform-scan", "bulk-scan", "rpe", "fact", "rpts", "am", "ia", "fis"].contains(product) || ["draft-store-service","div-cos","civil-general-applications", "civil-service", "civil-ccd","civil-camunda","civil-citizen-ui","civil-ga-ccd","rpe-service-auth-provider", "rpe-pdf-service", "cmc-citizen-frontend", "nfdiv-case-api", "nfdiv-frontend","prl-ccd-definitions","prl-cos","prl-dgs","prl-citizen-frontend","adoption-cos-api","adoption-web","ccd-case-disposer", "fpl-case-service", "fis-hmc-api", "fis-cos-api", "fis-ds-update-web", "fis-ds-web"].contains(appFullName) ? environment + ".platform.hmcts.net" : "service.core-compute-" + environment + ".internal"}'
```

Hi, you can keep this simpler and just keep fis - it will migrate every app for your product that way. The app one is an alternative in case you're not ready to move all of them over.
```suggestion
    zoneTemplate: '${["plum","probate","sscs","idam","ccd","ts","hmc","aac","rpo","camunda","dg","dm","dtsse","em","et","sptribs","nfdiv","pcq","lau","wa","finrem","jps","rd","xui", "reform-scan", "bulk-scan", "rpe", "fact", "rpts", "am", "ia", "fis"].contains(product) || ["draft-store-service","div-cos","civil-general-applications", "civil-service", "civil-ccd","civil-camunda","civil-citizen-ui","civil-ga-ccd","rpe-service-auth-provider", "rpe-pdf-service", "cmc-citizen-frontend", "nfdiv-case-api", "nfdiv-frontend","prl-ccd-definitions","prl-cos","prl-dgs","prl-citizen-frontend","adoption-cos-api","adoption-web","ccd-case-disposer", "fpl-case-service"].contains(appFullName) ? environment + ".platform.hmcts.net" : "service.core-compute-" + environment + ".internal"}'
```